docs-source/content/userguide/managing-fmw-domains/soa-suite/WebLogic-Logging-Exporter-Setup.md

```diff
@@ -8,17 +13,15 @@ This documents provides the steps to publish WebLogic Server logs into Elasticse
 
 ## Prerequisite
 
-This document assumes that you have already setup Elasticsearch/Kibana for logs collection. If you have not, please refer this [document](/kubernetes/samples/scripts/elasticsearch-and-kibana/README.md) to setup Elasticsearch/Kibana from WebLogic operator samples.
+This document assumes that you have already set up Elasticsearch and Kibana for logs collection. If you have not, please refer this [document](/kubernetes/samples/scripts/elasticsearch-and-kibana/README.md) to set up Elasticsearch and Kibana from WebLogic operator samples.
 
 ---  
 
-Please follow the below steps to push the server logs into Elasticsearch.  
-
 ## Download WebLogic Logging Exporter Binaries
```

Comment:
Download WebLogic Logging Exporter Binaries -> Download the WebLogic Logging Exporter binaries (use sentence capitalization in titles, globally)